### DISCUSSION TITLE
How about translations?

### DISCUSSION THREAD

**Discussion by DartPower** [diasurgical/DevilutionX#1557] [Ideas]
How about translations on other languages? Planned or not? *Hm*
e.g. Russian, Japanese, Italian, and etc... ?
Me just interesting.

**Comment by johndoe71rus** [comment]
example https://github.com/diasurgical/devilutionX/discussions/1229

and present branches translation, translation2

**Comment by qndel** [comment]
https://github.com/diasurgical/devilutionX/issues/66

**Comment by AJenbo** [comment]
the game can now be translated, we are currently working on adding fonts for more languages

**Reply by johndoe71rus** [reply]
How will you ensure that the received translation is not borrowed from another team from the past?

**Reply by AJenbo** [reply]
Same way as with all other content, same as other projects. I don't see how this is any different for translations.

**Comment by muzena** [comment]
@AJenbo 
How can I test translation?
I update hr.po, compile Diablo
Install diablo.deb file, but all text is on English.

**Reply by AJenbo** [reply]
atm they aren't part of the installation package. When saving in Poedit it will generate a .mo file that you need to put next to the application, then change the language code in the ini

**Comment by AAS** [comment]
@AJenbo Any updates on non latin chars?

**Reply by AJenbo** [reply]
Unfortunately resolving a lot of the capacity was soaked up by sorting out the Android situation. But I expect that there will be some progress on it in July.
One thing that has been done is cleaning up all the text rendering code which was an important first step. Still missing in adding UTF8->Unicode decoding and loading fonts from PCX files.

**Reply by NikoVP** [reply]
The unicode blocks that are currently rendered are the full: Basic Latin, Latin 1 Supplement, Latin Extended A, Latin Extended B, Coptic and Greek and Cyrillic. AJenbo already has them, so if anyone is willing to progress on that front, I am all for it 😊!
